app/assets/javascripts/covid19/src/webapp/hooks/useInfoDialog.ts

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import { Maybe } from "../../data/utils/ts-utils";
+import { sendAnalytics } from "../../utils/analytics";
+import { useBooleanState } from "./useBoolean";
+
+type UseInfoDialogReturn<T> = [
+    boolean,
+    () => void,
+    Maybe<T>,
+    (options: T, gaLabel: string) => void
+];
```

Comment:
No need to change it, but when the return array begins to grow, it may make sense to convert it to an object. Also, note that TS allows now to tag items in array: `type TaggedArray = [x: number, s: string]`. If keep the array, it will be better documented this way.